client/src/components/CustomAudioPlayer.css

```diff
@@ -0,0 +1,76 @@
+.custom-audio-player {
+  display: flex;
+  justify-content:center;
+  align-items: center;
+  width: 100%;
+  padding: 20px;
+}
+
+.stop-button,
+.play-button {
+  width: 58.209px;
+  height: 58.209px;
+  flex-shrink: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  padding-right: 0;
+  fill: #CB3100;
+}
+
+.stop-rectangles {
+  width: 13.92px;
```

Comment:
Is there a reason these px numbers need to be so precise?

Reply:
I copied from figma :)